actors/eam/src/lib.rs

```diff
@@ -267,10 +263,11 @@ impl EamActor {
 }
 
 impl ActorCode for EamActor {
```

Comment:
What if we just made the "builtin check" optional in the dispatch macro? It shouldn't be _too_ hard to add an `unrestricted` flag, or just duplicate the macro.

Reply:
I'm gonna hit an easy button and duplicate the macro, thanks.